account/domain.go

```diff
@@ -0,0 +1,15 @@
+package account
+
+type Account struct {
+	UserID  string `json:"user_id"`
+	ID      string `json:"id"`
+	Balance int    `json:"balance"`
+}
+
+type AccountResponse struct {
+	AccountInfo Account `json:"balance"`
+	Message     string  `json:"message"`
```

Comment:
wee dont need message field here

Reply:
instead write 2 separate methods to handle success and failure responses

methods should use following struct

type V2Resp struct {
	RequestID        string      `json:"request_id"`
	ErrorCode        int         `json:"error_code"`
	ErrorDescription string      `json:"error_description"`
	Data             interface{} `json:"data"`
}